examples/workflow/fan_out_fan_in.py

```diff
@@ -59,4 +57,4 @@ def process_results(ctx, final_result: int):
     print(f'Workflow started. Instance ID: {instance_id}')
     state = wf_client.wait_for_workflow_completion(instance_id)
 
-    workflowRuntime.shutdown()
+    wfr.shutdown()
```

Comment:
This can probably be done in another PR if you'd prefer, but one of the improvements we discussed was making it so that the runtime can be stopped automatically if we allow the `start()` method to be invoked using a [`with`](https://peps.python.org/pep-0343/) block.

Reply:
Will leave it for another PR.